tests/api/data_test.py

```diff
@@ -17,83 +15,12 @@ class TestAvailabilityState:  # pylint: disable=too-many-public-methods
     """
 
     @pytest.fixture
-    def session(self, session: Session) -> Session:
+    def client(self, client: TestClient, load_examples) -> TestClient:
         """
-        Add nodes to facilitate testing of availability state updates
+        load examples
         """
-        catalog = Catalog(name="test")
-        session.add(catalog)
-        session.commit()
-        session.refresh(catalog)
-
-        node1 = Node(
-            name="revenue_source",
-            type=NodeType.SOURCE,
-            current_version="1",
-        )
-        node_rev1 = NodeRevision(
-            name=node1.name,
-            node=node1,
-            version="1",
-            columns=[
-                Column(name="payment_id", type=ColumnType.INT),
-                Column(name="payment_amount", type=ColumnType.FLOAT),
-                Column(name="customer_id", type=ColumnType.INT),
-                Column(name="account_type", type=ColumnType.STR),
-            ],
-            tables=[
-                Table(
-                    database_id=1,
-                    catalog_id=catalog.id,
-                    schema="accounting",
-                    table="revenue",
-                ),
-            ],
-        )
-        node2 = Node(
-            name="large_revenue_payments_only",
-            type=NodeType.TRANSFORM,
-            current_version="1",
-        )
-        node_rev2 = NodeRevision(
-            name=node2.name,
-            node=node2,
-            version="1",
-            type=NodeType.TRANSFORM,
-            columns=[
-                Column(name="payment_id", type=ColumnType.INT),
-                Column(name="payment_amount", type=ColumnType.FLOAT),
-                Column(name="customer_id", type=ColumnType.INT),
-                Column(name="account_type", type=ColumnType.STR),
-            ],
-        )
-        node3 = Node(
-            name="large_revenue_payments_and_business_only",
-            type=NodeType.TRANSFORM,
-            current_version="1",
-        )
-        node_rev3 = NodeRevision(
-            name=node3.name,
-            node=node3,
-            version="1",
-            query=(
-                "SELECT payment_id, payment_amount, customer_id, account_type "
-                "FROM revenue_source WHERE payment_amount > 1000000 "
-                "AND account_type = 'BUSINESS'"
-            ),
-            type=NodeType.TRANSFORM,
-            columns=[
-                Column(name="payment_id", type=ColumnType.INT),
-                Column(name="payment_amount", type=ColumnType.FLOAT),
-                Column(name="customer_id", type=ColumnType.INT),
-                Column(name="account_type", type=ColumnType.STR),
-            ],
-        )
-        session.add(node_rev1)
-        session.add(node_rev2)
-        session.add(node_rev3)
-        session.commit()
-        return session
+        load_examples(client)
```

Comment:
This is a bit confusing, since in other tests we need to explicitly call `load_examples`, but here the `client` fixture calls it automatically. It would be better to be consistent in the fixtures so we don't have surprises. Maybe call this fixture `client_with_examples`?

Reply:
This is a great catch, I like the `client_with_examples`, updated this pattern to use that.